recurly/__init__.py

```diff
@@ -386,6 +386,13 @@ def create_shipping_address(self, shipping_address):
         url = urljoin(self._url, '/shipping_addresses')
         return shipping_address.post(url)
 
+class BillingInfoFraudInfo(recurly.Resource):
+    node_name = 'fraud_info'
```

Comment:
Wonder if this should be `node_name = 'fraud'` to echo [established pattern](https://github.com/recurly/recurly-client-python/blob/v2/recurly/__init__.py#L1394)

Reply:
This seems right to me -- same attributes and all. Is this the same fraud information denormalized into a different part of the response?